test/integration/features/basic.feature

```diff
@@ -145,7 +145,7 @@ Feature: Basic
      And executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" succeeds
      And executing "oc expose svc/ruby-ex" succeeds
      And executing "oc set probe dc/ruby-ex --readiness --get-url=http://:8080" succeeds
-     And service "ruby-ex" rollout successfully
+     And service "ruby-ex" rollout successfully in "1200" seconds
```

Comment:
Do we really need 20min for ruby app or you put this time out as a generic for all the rollout service ?

Reply:
We are probably good with 10 minutes, however this is a maximum time, if it takes less, then it will pass and continue with following steps.